### DISCUSSION TITLE
XP-Pen Deco 01 V2 Area Issue

### DISCUSSION THREAD

**Discussion by kujotakezo** [ppy/osu#23408] [Q&A] [answered]
When I try using my tablet (XP Pen Deco 01 V2) on lazer it detects an incorrect area. It only tracks a small area (about a quarter of the tablet) in the top left corner when selecting "full area" in the tablet settings (OpenTabletDriver has this exact same issue). The stock XP Pen drivers (pentablet) tracks the full area in windows just fine. From what I could see in a search nobody has posted about this issue before so I'm not sure if it's a bug or just an issue with my PC/tablet setup. Just posting to see if it's a bug or if there's a fix for this - I've attached my logs.

[updater.log](https://github.com/ppy/osu/files/11404827/updater.log)
[database.log](https://github.com/ppy/osu/files/11404829/database.log)
[input.log](https://github.com/ppy/osu/files/11404830/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11404831/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/11404832/network.log)
[performance.log](https://github.com/ppy/osu/files/11404833/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11404834/runtime.log)

**Comment by Theighlin** [comment, marked as answer]
Osu lazer uses OpenTabletDriver so you should report it to them if it happens in the standalone otd too.
